src/components/POLine/const.js

```diff
@@ -48,4 +48,11 @@ export const INVENTORY_RECORDS_TYPE = {
   none: 'None',
 };
 
+export const INVENTORY_RECORDS_TYPE_FOR_SELECT = [
+  { label: 'ui-orders.settings.createInventory.recordType.all', value: INVENTORY_RECORDS_TYPE.all },
```

Comment:
I think it should be `labelId` instead of `label`, and to be merged after https://github.com/folio-org/stripes-acq-components/pull/63

Reply:
`label` displayed directly, without translation. `labelId` is used for translation labels